src/class-options.php

```diff
@@ -25,14 +25,28 @@ class Options {
      **************************************************/
 
     public static function api_location() {
-        return getenv( 'WARPDRIVE_API' );
+        return self::env( 'WARPDRIVE_API' );
     }
 
     public static function access_token() {
-        return getenv( 'WARPDRIVE_ACCESS_TOKEN' );
+        return self::env( 'WARPDRIVE_ACCESS_TOKEN' );
     }
 
     public static function system_name() {
-        return getenv( 'WARPDRIVE_SYSTEM_NAME' );
+        return self::env( 'WARPDRIVE_SYSTEM_NAME' );
+    }
+
+    /**
+     * Wrapper around getenv()
+     * When not found with getenv() also check $_ENV and $_SERVER.
+     *
+     * @param $name
+     * @return array|false|mixed|string
+     */
+    private static function env($name) {
+        return (getenv($name) ?:
```

Comment:
Minor inconsistency, using `getenv` while using `self::env` in all other instances when reading environment variables.